src/viewer.js

```diff
@@ -3,6 +3,8 @@ let palette = require('google-palette');
 let _ = require('underscore')
 let Color = require('color');
 let slic = require('../addons/slic/slic');
+import React from 'react';
+import Annotations from './Annotations'
 
 var viewer = OpenSeadragon({
```

Comment:
Should this be importing from Viewer.jsx?

Reply:
actually, that file isn't used anymore. I kept it in the repo because we'll probably need to move more code out when we do the superpixels and classifier stuff.